### ISSUE TITLE
Using HDMI the Pi must reboot via selfmade kernel Panic from Default OS to recognize the VideoOutput

### ISSUE THREAD

**Issue by pierew** [maxnet/berryboot#67]
I must reput my sda1 for the OS's in the Pi to activate HDMI Output, the reput causes a Kernel Panic, that Reset the Pi

**Comment by maxnet** [comment]
Sorry, do not understand the issue. Can you rephrase it?

**Comment by pierew** [comment]
I've fixed it, I Must put on force_hdmi_hotplug - Because I not have seen the VideoOutput via HDMI, before i set this Configuration in config.txt
